test/current/BuildDotFhirDotOrgClient.test.ts

```diff
@@ -0,0 +1,459 @@
+                      'package-id': 'sushi-test-no-download',
+                      'ig-ver': '0.1.0',
+                      repo: 'sushi/sushi-test-no-download/branches/master/qa.json'
+                    },
+                    {
+                      url: 'http://hl7.org/fhir/sushi-test/ImplementationGuide/sushi-test-0.1.0',
+                      name: 'sushi-test',
+                      'package-id': 'sushi-test',
+                      'ig-ver': '0.1.0',
+                      repo: 'sushi/sushi-test/branches/master/qa.json'
+                    },
+                    {
+                      url: 'http://hl7.org/fhir/sushi-test-package-exists/ImplementationGuide/sushi-test-package-exists-0.1.0',
+                      name: 'sushi-test-package-exists',
+                      'package-id': 'sushi-test-package-exists',
+                      'ig-ver': '0.1.0',
+                      repo: 'sushi-no-manifest/sushi-test-package-exists/branches/master/qa.json'
+                    },
+                    {
+                      url: 'http://hl7.org/fhir/sushi-test-manifest-empty/ImplementationGuide/sushi-test-manifest-empty-0.1.0',
+                      name: 'sushi-test-manifest-empty',
+                      'package-id': 'sushi-test-manifest-empty',
+                      'ig-ver': '0.1.0',
+                      repo: 'sushi-manifest-empty/sushi-test-manifest-empty/branches/master/qa.json'
+                    },
+                    {
+                      url: 'http://hl7.org/fhir/sushi-test/ImplementationGuide/sushi-test-0.1.0',
+                      name: 'sushi-test',
+                      'package-id': 'sushi-test',
+                      'ig-ver': '0.2.0',
+                      repo: 'sushi/sushi-test/branches/testbranch/qa.json'
+                    },
+                  ]
+                };
+              } else if (
+                uri === 'https://build.fhir.org/ig/HL7/US-Core-R4/branches/main/package.manifest.json' ||
+                (uri.startsWith('https://build.fhir.org/ig/sushi/sushi-test') && uri.endsWith('json'))
+              ) {
+                return {
+                  data: {
+                    date: '20200413230227'
+                  }
+                };
+              } else if (
+                uri === 'https://build.fhir.org/ig/sushi-manifest-empty/sushi-test-manifest-empty/branches/master/package.manifest.json'
+              ) {
+                return {
+                  data: null
+                };
+
+              // is the not found case needed with new impl?
+              } else if (
+                uri === 'https://packages.fhir.org/hl7.fhir.r4b.core/4.1.0' ||
+                uri === 'https://packages.fhir.org/hl7.fhir.r5.core/4.5.0' ||
+                uri === 'https://packages.fhir.org/fhir.dicom/2021.4.20210910'
+              ) {
+                throw 'Not Found';
+              } else {
+                return {};
+              }
+            });
```

Comment:
I think this, and perhaps some other parts of this (?), is not used in the tests and can be removed.